README.md

```diff
@@ -43,8 +43,8 @@ The procedures after you submit the pull request is same as collaborators'. Read
 The estimated plan for this project is illustrated below
 - Sep. 24: Select base open-source project that will use for this project (**Finish**)
   + We Selected **Jekyll**
-- Oct. 05: Finish installing required environment (**On-going**)
-- Oct. 19: Designing Website / Obtain basic knowledge of the selected open-source project (**On-going**)
+- Oct. 04 ~~Oct. 05~~: Finish installing required environment (**Finish**)
+- Oct. 19: Designing Website / Obtain basic knowledge of the se-ected open-source project (**On-going**)
```

Comment:
I think it has a typo: not se-ected, but selected

Reply:
Typo Fixed